infor-design/enterprise
# 2744 - Fix uplift tab count style and refresh on theme switch.

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This issue fixes the tab styling on tab counts a bit and also changed the personalize and demo app logic to switch out the correct icons and refresh the page better.

Additionally added some notes on debugging travis and cleaned up lint on a few demo app files i looked at.

**Related github/jira issue (required)**:
Fixes #2743
Fixes #2744

**Steps necessary to review your pull request (required)**:

**Test One**
- go to http://localhost:4000/components/tabs/example-index.html
- select last tab on the right
- change theme to uplift (vibrant)
- the page should adjust the focus indicator

**Test Two**
- go to http://localhost:4000/components/tabs/example-counts.html
- select last tab on the right
- change theme to uplift (vibrant)
- the page should adjust the focus indicator

**Test Three**
- go to http://localhost:4000/components/tabs/example-counts.html
- has a bit more padding so things look a bit nicer on the tab count text

**Test Four**
- go to http://localhost:4000/components/icons/example-index.html
- switch back and forth between uplift  (vibrant) and soho (subtle) themes
- notice the actual icons are now changing

**Included in this Pull Request**:
